src/amber/cli/recipes/recipe_fetcher.cr

```diff
@@ -0,0 +1,72 @@
+require "http/client"
+require "zip"
+
+module Amber::Recipes
+  class RecipeFetcher
+    getter kind : String
```

Comment:
`kind` is opaque, but it might be the right name for the job. Adding a comment with an example to the definition here could help future maintainers.

Reply:
(Non Blocking) How many `kind` of recipes there are? If the list is short and known maybe converting it to an Enum would improve the intention.